client/scripts/views/components/new_thread_form.ts

```diff
@@ -134,42 +143,120 @@ export const NewThreadForm: m.Component<{
 }, {
   activeTag: OffchainTag | string,
   autoTitleOverride,
-  error,
   form: IThreadForm,
   fromDraft?: number,
   newType: string,
   quillEditorState,
+  recentlySaved: number[],
   saving: boolean,
   uploadsInProgress: number,
 }> = {
-  view: (vnode) => {
+  oninit: (vnode_) => {
+    console.log('initing');
+    console.log(vnode_.state.form);
```

Comment:
this can just be `vnode`, right?

also remove extra console.logs

Reply:
Linter throws a fit, but I can make a file-level exception

Reply:
oh, in that case it's fine, though i'm surprised that it does